src/toolchain-conventions.adoc

```diff
@@ -1,3 +1,17 @@
+== Target Triples
+
+The toolchains have agreed on the following architecture names to be used for
```

Comment:
I would prefer a different wording.
E.g. "The following architecture names are used for RISC-V in the first segment of target triplets".

Reply:
Sure, that sounds better.